light_gallery/templates/light_gallery.html

```diff
@@ -160,9 +185,28 @@
 {% endaddtoblock %}
 
 <div id="light-gallery-{{ id }}" class="djangocms-light-gallery {{ gallery_container_classes }}">
-  {% for image in images %}
-  <a href="{{MEDIA_URL}}{{ image.file }}" class="djangocms-light-gallery__link">
-    <img src="{% thumbnail image.file pageThumbWidthHeight upscale %}" class="djangocms-light-gallery__image">
-  </a>
-  {% endfor %}
+    {% for image in images %}
+        <div class="djangocms-light-gallery__outer-container">
+            <a href="{{MEDIA_URL}}{{ image.file }}" class="djangocms-light-gallery__link">
+                <div class="djangocms-light-gallery__image-container{% if thumbnail_orientation == 'h' %} djangocms-light-gallery__image-container--horizontal{% endif %}{% if thumbnail_orientation == 'hr' %} djangocms-light-gallery__image-container--horizontal-reverse{% endif %}">
+                    <img src="{% thumbnail image.file pageThumbWidthHeight upscale %}" class="djangocms-light-gallery__image{% if thumbnail_orientation == 'h' %} djangocms-light-gallery__image--horizontal{% endif %}{% if thumbnail_orientation != 'v' %} djangocms-light-gallery__image--horizontal{% endif %}{% if thumbnail_orientation == 'hr' %} djangocms-light-gallery__image--horizontal-reverse{% endif %}" alt="{{ image.default_alt_text }}">
```

Comment:
When there is no `default_alt_text` I'm seeing `None` displayed in the carousel.